cmd/kots/cli/install.go

```diff
@@ -698,7 +700,11 @@ func uploadAirgapArchive(deployOptions kotsadmtypes.DeployOptions, authSlug stri
 	if err != nil {
 		return false, errors.Wrap(err, "failed to add metadata")
 	}
-	if _, err := io.Copy(metadataPart, bytes.NewReader([]byte(deployOptions.License.Spec.AppSlug))); err != nil {
+	appSlug := ""
+	if deployOptions.License != nil {
```

Comment:
is there a need for these extra "not nil" checks if the original code didn't have them?